exporter/azuredataexplorerexporter/e2e_test.go

```diff
@@ -235,14 +231,7 @@ func TestCreateMetricsExporterE2E(t *testing.T) {
 	t.Cleanup(func() { _ = exp.Shutdown(context.Background()) })
 }
 
-func prepareQuery(tID string) (kusto.ParamTypes, kusto.QueryValues) {
-	kustoDefinitions := make(kusto.ParamTypes)
-	kustoParameters := make(kusto.QueryValues)
-	kustoDefinitions["TID"] = kusto.ParamType{Type: types.String}
-	kustoParameters["TID"] = tID
-	return kustoDefinitions, kustoParameters
-}
-
+// prepareQuery no longer used
```

Comment:
This seems like a review comment, not a code comment. Please remove

Reply:
Hello @dmitryax , Thanks for the comment. Fixed this and pushed the changes